tests/test_scraper.py

```diff
@@ -130,3 +135,320 @@ def test_save_image_file(self):
         ]
         assert self.tmpdir.join('img').join(
             expected_name).read('rb') == b'my.image.content'
+
+
+class GetImageNameTestCase(TestCase):
+
+    def test_get_image_name(self):
+        assert get_image_name(b'test_image.content') == hashlib.md5(
+            b'test_image.content').hexdigest() + '.jpg'
+
+
+class GetArticleAuthorDate(TestCase):
+
+    def test_get_article_author_date(self):
+        wiz_author_date = BeautifulSoup(
+            '''
+            <div class="dodano-2" style="float: left; 
+            margin-bottom: 1px;">Autor: <strong>Magdalena Nowicka-Franczak</strong> 
+            |  dodano: 2019-04-24</div>
+            '''.replace("\n", "")
```

Comment:
jak chcesz zrobić z tego jednolinijkowca to żeby nie stracić spacji na końcach linii polecam:
```python
wiz_author_date = BeautifulSoup(
    '<div class="dodano-2" style="float: left; ' 
    'margin-bottom: 1px;">Autor: <strong>Magdalena Nowicka-Franczak</strong> '
    '|  dodano: 2019-04-24</div>', 'html.parser')

```